.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels:
+      - 'dependencies'
+      - 'infrastructure'
```

Comment:
I don't think dependabot can do PEP 517 projects, but at least this will give us updates for GH Actions

Reply:
Cylc-flow isn't PEP517 yet but its deps are stored in the .cfg file, let's see how it gets on...